File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.cpp
class Solution {
public:
    bool isPalli(string word) {
        string newWord = word;
        reverse(newWord.begin(), newWord.end());
        return word == newWord;
    }

    string firstPalindrome(vector<string>& words) {
        for (string word : words) {
            if (isPalli(word)) {
                return word;
            }
        }
        return "";
    }
};

File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.java
class Solution {
    public boolean isPalli(String word) {
        String newWord = new StringBuilder(word).reverse().toString();
        return word.equals(newWord);
    }

    public String firstPalindrome(String[] words) {
        for (String word : words) {
            if (isPalli(word)) {
                return word;
            }
        }
        return "";
    }
}